app/src/main/kotlin/com/wire/android/ui/common/topappbar/CommonTopAppBar.kt

```diff
@@ -0,0 +1,134 @@
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.google.accompanist.systemuicontroller.rememberSystemUiController
+import com.wire.android.R
+import com.wire.android.ui.theme.wireColorScheme
+import com.wire.android.ui.theme.wireDimensions
+import com.wire.android.ui.theme.wireTypography
+
+@Composable
+fun CommonTopAppBar(commonTopAppBarViewModel: CommonTopAppBarViewModel) {
+
+    with (commonTopAppBarViewModel) {
+        returnToCall(
+            isMuted = callState.isMuted ?: false,
+            shouldShow = callState.shouldShow,
+            onClick = ::openOngoingCallScreen
+        )
+    }
+}
+
+@Composable
+private fun returnToCall(
+    isMuted: Boolean,
+    shouldShow: Boolean,
+    onClick: () -> Unit
+) {
+    if (shouldShow) {
+        val darkIcons = MaterialTheme.wireColorScheme.useDarkSystemBarIcons
+        rememberSystemUiController().setStatusBarColor(
+            color = MaterialTheme.wireColorScheme.callingReturnToCall,
+            darkIcons = darkIcons
+        )
+
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(28.dp)
+                .background(MaterialTheme.wireColorScheme.callingReturnToCall)
+                .clickable { onClick() },
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.Center
+        ) {
+            Row {
+                // Microphone Icon
+                Icon(
+                    painter = painterResource(
+                        id = if (isMuted) R.drawable.ic_microphone_white_muted
+                        else R.drawable.ic_microphone_white
+                    ),
+                    contentDescription = stringResource(
+                        id = if (isMuted) R.string.content_description_calling_call_muted
+                        else R.string.content_description_calling_call_unmuted
+                    ),
+                    tint = Color.White
+                )
+
+                // Camera Icon
+                Icon(
+                    painter = painterResource(id = R.drawable.ic_camera_white_paused),
+                    contentDescription = stringResource(R.string.content_description_calling_call_paused_camera),
+                    modifier = Modifier.padding(
+                        start = MaterialTheme.wireDimensions.spacing8x,
+                        end = MaterialTheme.wireDimensions.spacing8x
+                    ),
+                    tint = Color.White
```

Comment:
Could we use something from `ColorScheme` ? Will be helpful in later stages when we support dark mode :D